defaults/test_default.yaml

```diff
@@ -48,10 +48,10 @@ seeds_num: 1
 instance_provision: "spot"
 
 execute_post_behavior: false
-post_behavior_db_nodes: "keep-on-failure"
+post_behavior_db_nodes: "destroy"
 post_behavior_loader_nodes: "destroy"
 post_behavior_monitor_nodes: "keep-on-failure"
```

Comment:
Do we want to keep monitoring node too?

Reply:
The default here is a bit meaningless, since of the default on the pipelines.

It wouldn't affect local runs cause cleanup is a separate task (and people don't usually call it as in the pipe, so all the keep on failure logic doesn't kick-in)

Would be nice to align it with the pipeline, not more then that

Reply:
I don't want to change it, not here and not in the pipelines.
Only in some triggered pipelines where we clearly approve it.

Reply:
i don't think we should keep DB nodes by default, as no one is using them, and they are costing money without any reason... if someone wants to reproduce an issue, and intentionally keep them alive, this is something else, as the person will be owner of these instances, other than `timer` or `qa`, when automatic trigger is run